### DISCUSSION TITLE
Why in my setup I have to toggle manually in js dropdowns or popovers

### DISCUSSION THREAD

**Discussion by soinalastudio** [tabler/tabler#1096] [Q&A]
Hello,
Is there any methods to activate automatically dropdowns or popovers as per data attributes or to initialize all tabler stuffs?
For now, for dropdown for example, I have to add $("#buttonid").dropdown("toggle") in the button click event to toggle the dropdown menu.
For form-help popovers, I have to initialize manually $("#spanid").popover() so that popover will be triggered.
In tabler demo, I see that there is any call of those functions but they work...
Thank you much. I love tabler ui.

**Comment by rjd22** [comment]
@soinalastudio I'm not sure, this seems to be something on your end since it's working correctly in our demo and on my installation.

Are you sure you are running the included tabler.js? https://github.com/tabler/tabler/blob/main/dist/js/tabler.js

Without an error there is little that can be done to help you.
